test/Libraries/DynamoPythonTests/CodeCompletionTests.cs

```diff
@@ -117,7 +117,7 @@ public void CanMatchBasicNumVarSingleLine()
             Assert.IsTrue(matches.ContainsKey("a"));
             Assert.AreEqual("5.0", matches["a"]);
         }
-
+        
```

Comment:
Can we revert this file if there are no changes to it.

Reply:
Yes removing this file.